bindings/nodejs/lib/utils/utf8.ts

```diff
@@ -11,7 +11,7 @@ export const utf8ToBytes = (utf8: string) => {
 
 /** Convert hex encoded string to UTF8 string */
 export const hexToUtf8 = (hex: HexEncodedString) =>
-    decodeURIComponent(hex.replace('0x', '').replace(/[0-9a-f]{2}/g, '%$&'));
+    decodeURIComponent(hex.replace(/^0x/, '').replace(/[0-9a-f]{2}/g, '%$&'));
```

Comment:
Just curious, does // mean regex in TS ? Without quote or anything?

Reply:
i cannot find out :D seems to not be, i think it needs some info in between, but "///" is also taken as a string

Reply:
But this definitely works, right? No quote just seems super wild to me

Reply:
yep 100% wild stuff with nodejs guaranteed :D

Reply:
I doublechecked for ya, and this will turn into a RegExp type
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp

Reply:
Thanks